Library/Homebrew/cask/dsl.rb

```diff
@@ -175,6 +177,15 @@ def disabled? = @disabled
     sig { returns(T::Boolean) }
     def livecheck_defined? = @livecheck_defined
 
+    sig { void }
+    def set_no_autobump!
+      if @livecheck.strategy != :extract_plist
+        no_autobump! because: :extract_plist
+      elsif T.must(@version).latest?
```

Comment:
This seems like it will always fail. `@version` is set to `nil` just before this is called.